src/SFA.DAS.RoatpGateway.Domain/Apply/ApplyGatewayDetails.cs

```diff
@@ -7,6 +7,5 @@ public class ApplyGatewayDetails
         public DateTime? SourcesCheckedOn { get; set; }
         public string Comments { get; set; }
         public DateTime? OutcomeDateTime { get; set; }
-        public DateTime? ClarificationRequestedOn { get; set; }
```

Comment:
Is this not being used anymore?

Reply:
I figured the new audit stuff would be enough, as that tracks the status change and the updatedon.  Not sure now though.  I've raised this with you in slack channel dev

Reply:
Actually all I needed was the updatedon, which is already passed back as the field 'OutcomeMadeOn' from apply, so I didn't need it.  Re-requesting a review, as I've addressed all other issues.

Reply:
Cool